lib/SGN/Controller/AJAX/BrAPI.pm

```diff
@@ -3815,7 +3815,7 @@ sub observations_PUT {
 	my $version = $c->request->captures->[0];
 	my $brapi_package_result;
 	if ($version eq 'v2'){
-		my $force_authenticate = 0;
```

Comment:
I see the PUT force_authenticate was 0 before and the POST was 1 but now they will both be set to the same config value. Do these need to be independently controlled or was that just an oversight before?